CONTRIBUTING.md

```diff
@@ -0,0 +1,26 @@
+# Contributing guide
```

Comment:
Something basic to keep us going. I did not pushed `create_kafka.py` as it has tons of experiments